src/components/DrawerDashBoard.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+
+import Drawer from '@material-ui/core/Drawer';
+import BackArrowIcon from '@material-ui/icons/ArrowBack';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { ThemeProvider } from 'styled-components/macro';
+
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+
+import MetricCards from './MetricCardsContainer';
+import InformationCard from './InformationCard';
+import SiteDetail from './SiteDetail';
+import DropDown from './DropDown';
+import { theme } from './theme';
+import { ButtonStyle } from '../styles/MermaidStyledComponents';
+import { summary, histogram, drawerWidth } from '../constants/summary-information';
+
+import PropTypes from 'prop-types';
+
+const drawerStyleProperties = makeStyles(theme => ({
+  summaryDashboardProperty: {
+    paddingBottom: theme.spacing(8)
+  },
+  drawer: {
+    width: drawerWidth,
+    flexShrink: 0
+  },
+  drawerPaper: {
```

Comment:
Maybe add `overflow-x: hidden` within here to stop left/right scrolling along the x axis.